src/detail-page/components/Tabs.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { Route, Link, useRouteMatch } from 'react-router-dom';
+import { Card, Tabs as MuiTabs } from '@material-ui/core';
+import styled from 'styled-components';
+
+import { TabPanel } from './tab-components/TabPanel';
+import { Tab } from './tab-components/Tab';
+import { ResourcesTab } from './ResourcesTab';
+import { JobsTab } from './JobsTab';
+import { LearnTab } from './LearnTab';
+import { Category } from '../../data/TabCategories';
+
+const StyledTabs = styled(MuiTabs)`
+  border-bottom: 1px solid
+    ${props => props.theme.palette.text.disabled};
+  color: inherit;
+
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+`;
+
+const StyledCard = styled(Card)`
+  width: 100%;
+`;
+
+/* For now TabContainer receive props from its parent container
+with categories and learn, jobs and resources content.
+Because it isn't determined how they will look like it's type is unknown.
+It should be change when it will be determined */
+interface TabProps {
+  categories: Category[];
+  learn: unknown;
+  jobs: unknown;
+  resources: unknown;
+}
+
+export const Tabs: React.FC<TabProps> = ({
+  categories,
+  learn,
+  jobs,
+  resources,
+}) => {
+  const [chosenCategory, setChosenCategory] = useState(0);
+  const match = useRouteMatch();
+
+  const getTabContent = (categoryName: String) => {
+    switch (categoryName) {
+      case 'jobs':
+        return <JobsTab content={jobs} />;
+      case 'learn':
+        return <LearnTab content={learn} />;
+      case 'resources':
+        return <ResourcesTab content={resources} />;
+      default:
+        throw new Error('Wrong category name!');
+    }
+  };
+
+  return (
+    <StyledCard>
+      <StyledTabs indicatorColor="primary" value={chosenCategory}>
```

Comment:
It looks like we've had a few important changes to `master` meanwhile. Can we please update this branch (it should also allow us to display these new shiny features you added :) )?